recipes/imguizmo/all/conanfile.py

```diff
@@ -44,6 +43,12 @@ def source(self):
         extracted_dir = glob.glob("ImGuizmo-*/")[0]
         os.rename(extracted_dir, self._source_subfolder)
 
+    def requirements(self):
+        if self.version == "cci.20210223":
```

Comment:
Fragile but okay, as there are only 2 version available.

Reply:
I agree with you.
When imguizmo will bumpe 1.84, recipe should be modified.